src/index.js

```diff
@@ -1,6 +1,6 @@
 import './style.css';
 import Data from './modules/data.js';
-import { getItems, addToLocalStorage } from './modules/utility.js';
+import { getItems, addToLocalStorage, clearCompletedTasks } from './modules/utility.js';
```

Comment:
- [ ] After marking a task as completed, it is updated in the database 👏 However, if the page is reloaded, the checkbox of the task that is been marked as completed is no longer checked ☑️ even though `completed === true` in localstorage. Please fix this as it is very important for user experience.
![image](https://github.com/ClarenceAtim/ToDoList/assets/46839548/3a4f16a4-ea5a-4660-9125-88fbbb2304dc)